### PR TITLE
Synchronize files of DCIM folder

### PR DESCRIPTION
**Pull request by signedav**
On import copy files from DCIM folder to original project path and on export to offline project path.

Subfolders are created if they don't exist.
Existing files are overwritten.

Resolves ##93

### CODE REVIEW

File: qfieldsync/utils/file_utils.py [hunk 1]
```diff
@@ -104,3 +105,22 @@ def slugify(text: str) -> str:
     slug = re.sub(r'[-]+', '-', slug)
     slug = slug.lower()
     return slug
+
+
+def copy_images(source_folder, destination_folder):
+    if os.path.isdir(source_folder):
+        if not os.path.isdir(destination_folder):
+            os.mkdir(destination_folder)
```
Comment (m-kuhn) [inline qfieldsync/utils/file_utils.py]: Can we always assume the parent folder exists in every case?
I guess yes, but if not https://stackoverflow.com/a/600612/2319028

Comment (signedav) [reply qfieldsync/utils/file_utils.py]: In the context, yes. Since we only call this, when there is a project-folder.

**Comment by signedav** [comment]
@m-kuhn I inserted tests in the export test.
The import test is not yet implemented. Should I make a general import test in general or is it sufficient like this?

**Comment by signedav** [comment]
Okay to merge @m-kuhn 
We can improve the general export tests another time as well.

**Pull request merged**